### PR TITLE
Use kebab-case for bridge arguments

### LINKED ISSUE #851: Use kebab-case for bridge arguments
Follow up on #849.

Currently the `structop::clap::arg_enum!` macro generates `FromStr` and `Display` + `fn variants` for an enum that represents a chain pair (a bridge). However the enum simply uses variant name for it's stringified version, so:
```rust
enum Bridge {
  RialtoToWestend,
  RialtoToMillau,
}
```
Leads to CLI arguments that look like that: `init-bridge RialtoToWestend` instead of `init-bridge rialto-to-westend`.

We rather prefer the latter, but without a need to manually implement anything. We should consider either:
1. Using some proc-macro libraries to get what we want (`derive_more`?, `serde`?)
2. Implement simple macro that does that for us.

### PR DESCRIPTION
**Pull request by halfpint104**
This PR closed #851. Changing CLI arguments for message sending, bridge initialisation and header relaying to be kebab case.

### CODE REVIEW

**Review by tomusdrw — APPROVED** [review]

**Comment by tomusdrw** [comment]
Awesome! Thanks a lot!

File: relays/bin-substrate/Cargo.toml [hunk 1]
```diff
@@ -17,6 +17,7 @@ num-format = "0.4"
 num-traits = "0.2"
 paste = "1.0"
 structopt = "0.3"
+strum = { version = "0.21.0", features = ["derive"] }
```
Comment (HCastano) [inline relays/bin-substrate/Cargo.toml:20]: Why did you choose `strum` over `serde` or `derive_more`?

**Review by HCastano — APPROVED** [review]
Thanks!

**Pull request merged**